src/pages/steps/PriceInformation.tsx

```diff
@@ -0,0 +1,97 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { parseSpacing } from '@/ui/Box';
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Icon, Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
+import { Stack } from '@/ui/Stack';
+import { Text } from '@/ui/Text';
+import { getValueFromTheme } from '@/ui/theme';
+
+import { Rate } from './modals/PriceInfoModal';
+
+const getValue = getValueFromTheme('priceInformation');
+
+type Props = {
+  priceInfo: Rate[];
+  onClickAddPriceInfo: () => void;
+  onClickAddFreePriceInfo: () => void;
+};
+
+const PriceInformation = ({
+  priceInfo,
+  onClickAddPriceInfo,
+  onClickAddFreePriceInfo,
+}: Props) => {
+  const { t, i18n } = useTranslation();
+
+  const hasPriceInfo = useMemo(() => {
+    return priceInfo.length > 0;
+  }, [priceInfo]);
```

Comment:
I wouldn't use `useMemo` here because getting `priceInfo.length` is a really fast access with no computation required. If you want to keep the `hasPriceInfo` it can just be a normal function